assets/js/components/product-category-control/index.js

```diff
@@ -21,19 +21,21 @@ class ProductCategoryControl extends Component {
 		super( ...arguments );
 		this.state = {
 			list: [],
+			loading: true,
 		};
 		this.renderItem = this.renderItem.bind( this );
 	}
 
 	componentDidMount() {
+		this.setState( { loading: true } );
```

Comment:
Will this  trigger a re-render immediately after mount? And if so - do we need to do this since `loading` is already set to `true` in the initial state?

Reply:
Tested it with this line commented out, still seems to render the same amount of times 🤷‍♂️

Reply:
Pretty sure react is smart enough to merge the state & not re-render. I think you're right about it not being necessary since it's set in the initial state... I think I'll remove it before merging.